llvm/test/Analysis/LoopAccessAnalysis/non-constant-distance-backward.ll

```diff
@@ -72,38 +77,97 @@ exit:
 }
 
 define void @backward_min_distance_120(ptr %A, i64 %N) {
-; COMMON-LABEL: 'backward_min_distance_120'
-; COMMON-NEXT:    loop:
-; COMMON-NEXT:      Memory dependences are safe with run-time checks
-; COMMON-NEXT:      Dependences:
-; COMMON-NEXT:      Run-time memory checks:
-; COMMON-NEXT:      Check 0:
-; COMMON-NEXT:        Comparing group ([[GRP3:0x[0-9a-f]+]]):
-; COMMON-NEXT:          %gep.off.iv = getelementptr inbounds i8, ptr %gep.off, i64 %iv
-; COMMON-NEXT:        Against group ([[GRP4:0x[0-9a-f]+]]):
-; COMMON-NEXT:          %gep = getelementptr inbounds i8, ptr %A, i64 %iv
-; COMMON-NEXT:      Grouped accesses:
-; COMMON-NEXT:        Group [[GRP3]]:
-; COMMON-NEXT:          (Low: {(15 + %A)<nuw>,+,1}<nuw><%outer.header> High: {(271 + %A),+,1}<nw><%outer.header>)
-; COMMON-NEXT:            Member: {{\{\{}}(15 + %A)<nuw>,+,1}<nuw><%outer.header>,+,1}<nuw><%loop>
-; COMMON-NEXT:        Group [[GRP4]]:
-; COMMON-NEXT:          (Low: %A High: (256 + %A))
-; COMMON-NEXT:            Member: {%A,+,1}<nuw><%loop>
-; COMMON-EMPTY:
-; COMMON-NEXT:      Non vectorizable stores to invariant address were not found in loop.
-; COMMON-NEXT:      SCEV assumptions:
-; COMMON-EMPTY:
-; COMMON-NEXT:      Expressions re-written:
-; COMMON-NEXT:    outer.header:
-; COMMON-NEXT:      Report: loop is not the innermost loop
-; COMMON-NEXT:      Dependences:
-; COMMON-NEXT:      Run-time memory checks:
-; COMMON-NEXT:      Grouped accesses:
-; COMMON-EMPTY:
-; COMMON-NEXT:      Non vectorizable stores to invariant address were not found in loop.
-; COMMON-NEXT:      SCEV assumptions:
-; COMMON-EMPTY:
-; COMMON-NEXT:      Expressions re-written:
+; CHECK-LABEL: 'backward_min_distance_120'
+; CHECK-NEXT:    loop:
+; CHECK-NEXT:      Memory dependences are safe with a maximum safe vector width of 120 bits
+; CHECK-NEXT:      Dependences:
+; CHECK-NEXT:        BackwardVectorizable:
+; CHECK-NEXT:            %l = load i8, ptr %gep, align 4 ->
+; CHECK-NEXT:            store i8 %add, ptr %gep.off.iv, align 4
+; CHECK-EMPTY:
+; CHECK-NEXT:      Run-time memory checks:
+; CHECK-NEXT:      Grouped accesses:
+; CHECK-EMPTY:
+; CHECK-NEXT:      Non vectorizable stores to invariant address were not found in loop.
```

Comment:
Can keep common here as well?

Reply:
How? VW128 and MAXLEN have runtime checks, while CHECK doesn't.